tests_e2e/tests/lib/network_security_rule.py

```diff
@@ -55,7 +55,7 @@ def add_security_rule(self, security_rule: Dict[str, Any]) -> None:
         self._get_network_security_group()["properties"]["securityRules"].append(security_rule)
 
     def _get_network_security_group(self) -> Dict[str, Any]:
-        resources: List[Dict[str, Any]] = self._template["resources"]
+        resources: Dict[str, Dict[str, Any]] = self._template["resources"]
```

Comment:
Schema updated in new lisa template; resources type changed to dict(dict) from list(dict)